.pipelines/prod-release.yml

```diff
@@ -65,7 +70,10 @@ stages:
     parameters:
       environment: RP-Prod-EuropeSector
       locations:
+      - canadacentral
+      - canadaeast
```

Comment:
You don't like canada? Trying to move it to europe :smile:

Reply:
We will take it

Reply:
Always wanted to visit Canada. Ryanair, here I come.

Reply:
In v3.11 we have Canada in low traffic and Japan in row. But this should be fine